website/source/docs/configuration/storage/raft.html.md

```diff
@@ -44,7 +44,9 @@ cluster_addr = "http://127.0.0.1:8201"
 
 - `path` `(string: "")` – The file system path where all the Vault data gets
   stored.
+  This value can be overridden by setting the `VAULT_RAFT_PATH` environment variable.
 
 - `node_id` `(string: "")` - The identifier for the node in the Raft cluster.
+  This value can be overridden by setting the `VAULT_RAFT_NODE_ID` environment variable.
```

Comment:
The documentation states the environment variable overrides the config setting, but the code prioritizes the config. Could you switch the code so the environment is checked first and only if it's an empty string do we read from the config map?

Reply:
Nice catch! In the case of Kubernetes the config file would probably not contain any values but this makes sense. I changed it – have another look?